schema.prisma

```diff
@@ -0,0 +1,71 @@
+datasource db {
+  url      = env("DATABSE_URL")
+  provider = "sqlite"
+}
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+model User {
+  id         Int         @id @default(autoincrement())
+  createdAt  DateTime    @default(now())
+  email      String      @unique
+  fName      String
+  lname      String
+  scoreCards ScoreCard[]
+}
+
+model Course {
+  id         Int      @id @default(autoincrement())
+  name       String
+  location   Location @relation(fields: [locationId], references: [id])
+  locationId Int
+  layouts    Layout[]
+}
+
+model Location {
+  id      Int      @id @default(autoincrement())
+  name    String
+  lat     Float
+  lon     Float
+  courses Course[]
```

Comment:
Any reason this is a 1-to-many relationship between courses and location? My assumption would be 1-to-1 with course

Reply:
Changed, it added them as 1 to many when it formatted on save as prisma either requires 1 to many of the possibility of null vlaues